### PR TITLE
fix: show the user that has created that particular user

### LINKED ISSUE #2285: User management
In user list page (http://care.mn.gov.in/users) , on each user card, also show the user that has created that particular user

### PR DESCRIPTION
**Pull request by Pragati1610**
On the user list page, for each user
- [x] show the user that has created that particular user

fixes #2285

### CODE REVIEW

**Comment by developedBySJ** [comment]
@Pragati1610 Add a screenshot when you create a pull request

File: src/Components/Common/UserDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+function UserDetails(props: { children: React.ReactNode; title: string }) {
```
Comment (developedBySJ) [inline src/Components/Common/UserDetails.tsx:3]: Instead of passing label through children 
- Add `label` prop of type `string`

**Review by developedBySJ — CHANGES_REQUESTED** [review]

File: src/Components/Common/UserDetails.tsx (hunk 1 above)
Comment (Pragati1610) [reply src/Components/Common/UserDetails.tsx:3]: `title` corresponds to `label` in the UserDetails component. I'm using the children prop for the data corresponding to that label like this : 

```typescript
<UserDetails title="District">
    <div className="font-semibold">
    	{user.district_object.name}
    </div>
<UserDetails>
```

Comment (Pragati1610) [reply src/Components/Common/UserDetails.tsx:3]: based on this @developedBySJ, is the change still needed?

**Comment by nihal467** [comment]
@Pragati1610 @developedBySJ @gigincg is it good for testing

**Comment by Pragati1610** [comment]
> @Pragati1610 @developedBySJ @gigincg is it good for testing

Yes, backend PR has been merged

**Comment by nihal467** [comment]
@Pragati1610 @khavinshankar what is the status of the PR

**Comment by Pragati1610** [comment]
> @Pragati1610 @khavinshankar what is the status of the PR

Could you take a look at that GitGuardian comment for me once?
It shows a secret, RECAPTCHA_SITE_KEY has been disclosed, but afaik, it was there from before
So should it be removed? and if yes, then how should we access the env var?

**Comment by Pragati1610** [comment]
This PR can be removed. #2435 has already been successfully closed, the same code can be found in the PR for that issue and it  has already been merged

**Pull request closed without merging**